cli/src/main.rs

```diff
@@ -167,6 +167,26 @@ fn codegen<I: Input>(
     derives.append(p.into_iter());
 
     let runtime_api = generator.generate_runtime(item_mod, derives);
-    println!("{}", runtime_api);
+    println!(
+        "
+        // Copyright 2019-2022 Parity Technologies (UK) Ltd.
+        // This file is part of subxt.
+        //
+        // subxt is free software: you can redistribute it and/or modify
+        // it under the terms of the GNU General Public License as published by
+        // the Free Software Foundation, either version 3 of the License, or
+        // (at your option) any later version.
+        //
+        // subxt is distributed in the hope that it will be useful,
+        // but WITHOUT ANY WARRANTY; without even the implied warranty of
+        // MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+        // GNU General Public License for more details.
+        //
+        // You should have received a copy of the GNU General Public License
+        // along with subxt.  If not, see <http://www.gnu.org/licenses/>.
```

Comment:
Actually, I don't reckon we can/should claim any copyright over the codegen output (it is all theirs, and it's up to them to then do whatever they like with it I'd say).